packages/components/src/presentation-components/tree/controlled/PresentationTreeRenderer.tsx

```diff
@@ -42,12 +41,11 @@ export interface PresentationTreeRendererProps extends TreeRendererProps {
  * Return type of [[useFilterablePresentationTree]] hook.
  * @beta
  */
-export interface FilterableTreeProps<T extends HTMLElement> {
+export interface FilterableTreeProps {
   onFilterClick: (nodeId: string) => void;
   onClearFilterClick: (nodeId: string) => void;
   /** Reference of the document body. Needs to be passed to the root of the tree for the filter dialog to be placed separately from the tree context. */
```

Comment:
should the comment be updated or removed?

Reply:
removed

Reply:
I don't see the comment being removed.